deploy/pg2sphinx_trigger.py

```diff
@@ -215,72 +235,99 @@ def pg_get_tables(sql_query,sql_db):
                         INSERT INTO indexes (
                             sphinx_index
                             , index_parent
-                            ,source
+                            , source
                             ) VALUES(?, ?, ?);""" ,(index.strip(), str(index_parent).strip(), source))
 
     # output
     sql = """
-        select
+        SELECT
             a.source as source
-            , coalesce(a.sql_db,b.sql_db) as database
+            , coalesce(a.sql_db, b.sql_db) as database
             , a.sql_query as sql
             , i.sphinx_index
             , i.index_parent
-            FROM 
-            indexes i left join indexes p on trim(i.index_parent)=trim(p.sphinx_index)
-            left join sources a on coalesce(i.source,p.source) = a.source
-            left join sources b on a.source_parent = b.source
+            FROM
+            indexes i LEFT JOIN indexes p on trim(i.index_parent) = trim(p.sphinx_index)
+            LEFT JOIN sources a on coalesce(i.source, p.source) = a.source
+            LEFT JOIN sources b on a.source_parent = b.source
     """
-    
-    resultat=[]
+
+    tables_target = []
+    db_path = options.database_filter.split('.')
+    is_table_target = bool(re.match('^[a-z]*_(dev|int|prod).[a-z]*.[a-z]*$', options.database_filter))
+    if is_table_target:
+        sql_target = 'SELECT * FROM %s.%s' % (db_path[1], db_path[2])
+        try:
+            tables_target = pg_get_tables(sql_target, db_path[0])
```

Comment:
this will return you only the table itself won't it?
pg_get_tables is only making sense when you would like to get the list of tables behind views or cascaded views.